qm-plugins/qm-object-cache/html/class-qm-output-html-object-cache-group-stats.php

```diff
@@ -166,11 +166,11 @@ public function output_sortable_table_col( string $title ) {
 	 * Outputs a table cell.
 	 *
 	 * @param string $value Value to be outputted in table cell
-	 * @param int|null $weight Weight by sorting priority
+	 * @param float|null $weight Weight by sorting priority
 	 *
 	 * @return void
 	 */
-	public function output_table_cell( ?string $value, int $weight = null ) {
+	public function output_table_cell( ?string $value, ?float $weight = null ) {
```

Comment:
If `$weight` is `float` now, shouldn't we explicitly convert it to `int` when printing the `data-qm-sort-weight` attribute?

Reply:
Its my understanding that this data attribute is used for sorting the table in JS. I don't see any issues with leaving it as a float for display purposes. I'd argue it adds a higher level of accuracy to the sort, but I doubt that adds up to any difference realistically.